Documentation/ceph-object-bucket-claim.md

```diff
@@ -105,13 +105,14 @@ provisioner: rook-ceph.ceph.rook.io/bucket [2]
 parameters: [3]
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
-  bucketName: ceph-bucket [4]
-reclaimPolicy: Delete [5]
+  region: us-west-1 [4]
+  bucketName: ceph-bucket [5]
+reclaimPolicy: Delete [6]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
 1. **all** `parameter` required.
+1. `region` needs to create required zone/zonegroup in RGW before defining it here. Please refer [ceph documentation](https://docs.ceph.com/en/latest/radosgw/multisite/#multi-site) for more details.
```

Comment:
Where in the doc? The current description is not really clear to me.

Reply:
There is mention in the doc `zonegroup` is initially known as region(no reference to aws here). I have checked with team and in the ceph doc we need to make it explicit as well. For the time being, I will point to `zonegroup`  link